index.js

```diff
@@ -50,9 +50,8 @@ module.exports = function (name, version) {
 		})
 		.catch(function (err) {
 			if (err.statusCode === 404) {
-				throw new Error('Package doesn\'t exist');
+				throw new Error('Package "' + name + '" doesn\'t exist');
 			}
-
```

Comment:
Just a tip. I always look through the diff before submitting a PR. That has caught a lot of mistakes in the past.